SparkMonitor/sparkmonitor/kernelextension.py

```diff
@@ -186,7 +187,10 @@ def configure(conf):
     log.info(os.environ["SPARKMONITOR_KERNEL_PORT"])
     conf.set("spark.extraListeners",
              "sparkmonitor.listener.JupyterSparkMonitorListener")
-    jarpath = os.path.abspath(os.path.dirname(__file__)) + "/listener.jar"
+    if "2.11" in get_spark_version():
```

Comment:
seems a bit unnecessary to execute a command 2 times (we should store the version result in a var).
Also, there's no else, so this might crash?

Reply:
good poing, adding variable. today all of spark is with scala 11 or 12, even otherwise it doesn't crash, extension has nothing to show